staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -28,6 +28,8 @@ const (
 	labelPrefix = "batch.kubernetes.io/"
 
 	JobCompletionIndexAnnotation = labelPrefix + "job-completion-index"
+	// JobCompletionIndexLabel is defined following the format <controller>.kubernetes.io/pod-index
```

Comment:
I think you need to replicate this here:

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/batch/types.go#L25

Reply:
I don't see the JobCompletionIndexAnnotation there though, seems like it doesn't include everything, do you know why that is?

Reply:
It is not necessary to put it in the internal api file unless the api-server needs to reference it.